content/en/contributing/multi-account-region-testing.md

```diff
@@ -0,0 +1,73 @@
+
+
+## Tips to make your changes compatible with multi-accounts and regions
+
+In order to make your changes compatible with multi-accounts and regions, you should follow the below tips:
+
+- For cross accounts, in inter service communication for many integrations, you can specify a role, with which permissions the source service makes a request to the target service, to access other services resource. This role should be in source account. When writing an AWS validated test case, you need to properly configure IAM roles. 
+
+    For example: 
+    In the test case `test_api_gateway_kinesis_integration` we have specified a [role](https://github.com/localstack/localstack/blob/ae31f63bb6d8254edc0c85a66e3c36cd0c7dc7b0/tests/aws/services/apigateway/test_apigateway_basic.py#L2017-L2022) in the apigateway which has permissions to access the target kinesis account.
+    ```python
+    result = self.connect_api_gateway_to_kinesis(
+            client,
+            api_name,
+            stream_name,
+            region_name,
+            role_arn,
+        )
+    ```
+
+- While having an inter service communication in cross-accounts, you can create the client using `with_assumed_role(…)`:
+
+    For example:
+    ```python
+    result = self.connect_api_gateway_to_kinesis(
+            client,
+            api_name,
+            stream_name,
+            region_name,
+            role_arn,
+        )
+    ```
+    
+- When there is no role specified, you should use the source arn conceptually if cross-account is allowed. 
+
+    For example:
+    This can be seen in a case where we [added](https://github.com/localstack/localstack/blob/ae31f63bb6d8254edc0c85a66e3c36cd0c7dc7b0/localstack/utils/aws/message_forwarding.py#L42) `account_id` to [send events to the target](https://github.com/localstack/localstack/blob/ae31f63bb6d8254edc0c85a66e3c36cd0c7dc7b0/localstack/utils/aws/message_forwarding.py#L31) service like sqs, sns, lambda, etc. 
+
+- You should always refer the official AWS API docs and check how the the application is communicating to each other. 
+    
+    For example: 
+    Firehose to s3 refer to [Offical AWS Docs](https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html#cross-account-delivery-s3).
+
+
+## Manually trigger the workflow
+
+To manually trigger the scheduled workflow in case you have the permissions, you can change set the value of `randomize-aws-credentials` to `true` in the [workflow](https://github.com/localstack/localstack/blob/master/.circleci/config.yml#L13). This will trigger the tests with randomised account and region credentials.
```

Comment:
Please describe here how you can trigger the workflow with the proper parameters in the UI, and not with a commit.
Also, this line contains multiple sentences.